vassal-app/src/main/resources/VASSAL/i18n/Editor.properties

```diff
@@ -272,7 +272,8 @@ Editor.MouseOverStackViewer.non_stacking=Include non-stacking pieces?
 Editor.MouseOverStackViewer.move_selected=Include move-when-selected pieces?
 Editor.MouseOverStackViewer.non_moveable=Include non-movable pieces?
 Editor.MouseOverStackViewer.unrotated_state=Show pieces in un-rotated state?
-Editor.MouseOverStackViewer.top_deck=Include top piece in Deck?
+Editor.MouseOverStackViewer.show_deck=Show top pieces of a Deck
```

Comment:
Since all the other related options say "Include..." I suggest changing "Show" to "Include" in this show_deck message too, just for additional clarity that they are all related options. It's fine to leave "show" in the show_deck_depth message though, I think that works well.